docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -88,22 +88,34 @@ Use this to orient the ADC to determine the location to insert your wires.
 {{% /alert %}}
 
 Insert the MCP3008 into your breadboard so that it bridges both sides of the divide.
-Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your machine to connect the pins with wires as follows:
+Now you can use the breadboard points adjacent to the MCP3008 to connect pins on the MCP3008 to your Raspberry Pi and other peripherals using jumper wires.
```

Comment:
```suggestion
Now you can use the breadboard points next to the MCP3008 to connect pins on the MCP3008 to your Raspberry Pi and other peripherals using jumper wires.
```
simple language >

[opt] Do we need to explain how a breadboard works here or can this be shorter?

Reply:
I'm trying to keep it accessible even for folks who aren't that familiar with breadboards!